lib/http/HttpClient_WinInet.cpp

```diff
@@ -136,6 +139,105 @@ class WinInetRequestWrapper
         return true;
     }
 
+    DWORD GetDWORDRegKey(HKEY hKey, const std::string& subKey, const std::string& value) {
```

Comment:
If these are always known at build time, don't do unnecessary heap allocations, use const char* rather than std::string

Reply:
std::string_view would be ideal, but this repo is still not on C++17 for some reason.